library/accessory_workflows/build_cellranger_reference/BuildCellRangerReferenceBundle.wdl

```diff
@@ -0,0 +1,80 @@
+task BuildCellRangerReference {
+  String ref_fasta
+  String gtf_file
+  String ref_name
+
+  String docker = "quay.io/humancellatlas/secondary-analysis-cellranger"
+  String memory = "416 GB"
+  Int boot_disk_size_gb = 12
+  String disk_space = "250"
+  Int cpu = 64
+
+  command {
+    set -e
+
+    wget ${ref_fasta}
+    ref_fasta_tar_filename=$(echo ${ref_fasta} | sed 's:.*/::')
+    gunzip $ref_fasta_tar_filename
+
+    ref_fasta_filename=$(echo "$ref_fasta_tar_filename" | rev | cut -c4- | rev)
+
+    wget ${gtf_file}
+    gtf_tar_filename=$(echo ${gtf_file} | sed 's:.*/::')
+    gunzip $gtf_tar_filename
+
+    gtf_filename=$(echo "$gtf_tar_filename" | rev | cut -c4- | rev)
```

Comment:
I would use the pattern above for the fasta for the gtf as well. If this is notably slow, you can probably perform the two commands in parallel.